platform/ios/src/MGLMapView.mm

```diff
@@ -4605,7 +4606,7 @@ - (void)updateHeadingForDeviceOrientation
     return [self visibleFeaturesAtPoint:point inStyleLayersWithIdentifiers:nil];
 }
 
-- (NS_ARRAY_OF(id <MGLFeature>) *)visibleFeaturesAtPoint:(CGPoint)point inStyleLayersWithIdentifiers:(NS_SET_OF(NSString *) *)styleLayerIdentifiers
+- (NS_ARRAY_OF(id <MGLFeature>) *)visibleFeaturesAtPoint:(CGPoint)point inStyleLayersWithIdentifiers:(NS_SET_OF(NSString *) *)styleLayerIdentifiers withPredicate:(NSPredicate *)predicate
```

Comment:
Replace `withPredicate:` with `predicate:`. While it can be helpful to make a selector read like an English sentence, at a certain point that falls apart and it’s better to name the rest of the parameters without prepositions.